rudra/data_store/bigquery/base.py

```diff
@@ -44,11 +33,53 @@ def __init__(self, query_job_config=None, credential_path=None):
 
         self.client = None
 
-        if self.new_credential_path:
+        if self.credential_path:
             self.client = bigquery.Client(
                 default_query_job_config=self.query_job_config)
         else:
             raise ValueError("Please provide the the valid credential_path")
+        tfile.close()
```

Comment:
The file will get deleted at this point. I hope that's fine, right?

Reply:
Yep just tested it! seems to be working fine since the BQ object is already authenticated and created, will check it once more for the other ecosystems and merge it if it looks good.